index.js

```diff
@@ -50,6 +52,8 @@ class Conf {
 		} catch (e) {
 			this.store = store;
 		}
+
+		this.migrate(options, options.projectVersion);
```

Comment:
I think you should do the if conditional here instead of inside `migrate()`.

Reply:
Passing in `options.projectVersion` seems moot as you're already passing in the `options` object.